### PR TITLE
Feat/map 69/add swagger for the backend add basic configuration

### PR DESCRIPTION
**Pull request by mazursasha1990**

### CODE REVIEW

File: backend/README.md [hunk 1]
```diff
@@ -19,3 +19,4 @@ To run backend, you can use **docker** installed on your machine as well.
 [Server on which backend located](http://3.126.245.53:3001)
 
 _TODO: Add Swagger_
```
Comment (steadyGuy) [inline backend/README.md]: remove todo

File: backend/index.ts [hunk 2]
```diff
@@ -30,13 +31,30 @@ app.use(cors({ credentials: true, origin: "*" }));
...
+  definition: {
+    openapi: "3.0.0",
+    info: {
+      title: "Library API",
+      version: "1.0.0",
+      description: "MapUA API",
+    },
+    servers: [
+      {
+        url: "http://localhost:3001",
```
Comment (steadyGuy) [inline backend/index.ts]: Create a varaibale HOST inside .env file (and add this variable to .envExample file) and assign http://localhost:3001 to this variable
then use process.env.HOST

File: backend/package.json [hunk 3]
```diff
@@ -14,12 +14,14 @@
     "@types/cors": "^2.8.12",
     "@types/express": "^4.17.13",
     "@types/node": "^17.0.19",
+    "@types/swagger-jsdoc": "^6.0.1",
```
Comment (steadyGuy) [inline backend/package.json]: put it in devDependencies

File: package.json [hunk 4]
```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "@types/swagger-ui-express": "^4.1.3"
+  }
+}
```
Comment (steadyGuy) [inline package.json]: Remove this file.

File: yarn.lock [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+  integrity sha512-nCkHGI4w7ZgAdNkrEu0bv+4xNV/XDqW+DydknebMOQwkpDGx8G+HTlj7R7ABI8i8nKxVw0wtKPi1D+lPOkh4YQ==
+  dependencies:
+    "@types/mime" "^1"
+    "@types/node" "*"
+
+"@types/swagger-ui-express@^4.1.3":
+  version "4.1.3"
+  resolved "https://registry.yarnpkg.com/@types/swagger-ui-express/-/swagger-ui-express-4.1.3.tgz#7adbbbf5343b45869debef1e9ff39c9ba73e380f"
+  integrity sha512-jqCjGU/tGEaqIplPy3WyQg+Nrp6y80DCFnDEAvVKWkJyv0VivSSDCChkppHRHAablvInZe6pijDFMnavtN0vqA==
+  dependencies:
```
Comment (steadyGuy) [inline yarn.lock]: remvoe this file

**Review by steadyGuy — CHANGES_REQUESTED** [review]
Remove node_modules folder, which located on top of the root foled. _package.json_ and _yarn.lock_ remove too.

**Review by steadyGuy — CHANGES_REQUESTED** [review]
you didn't remove files that I had mentioned 
image

**Review by steadyGuy — CHANGES_REQUESTED** [review]
Remove node_modules from the top root.
image

**Review by steadyGuy — APPROVED** [review]

**Review by romawht22 — APPROVED** [review]

**Pull request merged**